Example/Tests/BONURLTestCase.m

```diff
@@ -0,0 +1,47 @@
+//
+//  BONURLTestCase.m
+//  BonMot
+//
+//  Created by Phil Larson on 6/24/16.
+//  Copyright © 2016 Zev Eisenberg. All rights reserved.
+//
+
+#import "BONBaseTestCase.h"
+
+@import BonMot;
+
+@interface BONURLTestCase : BONBaseTestCase
+
+@end
+
+@implementation BONURLTestCase
+
+- (void)testNeither
+{
+    NSString *quote = @"Hello, world";
+    NSDictionary *attributes = BONChain.new.string(quote).attributes;
+    
+    XCTAssertNil(attributes[NSLinkAttributeName]);
+}
+
+- (void)testLink
+{
+    NSAttributedString *attributedString =
+    BONChain.new
+    .string(@"Hello, world")
+    .url([NSURL URLWithString:@"https://apple.com"])
+    .attributedString;
+    
+    XCTAssertEqualObjects(attributedString.string, @"Hello, world");
+
+    NSDictionary *controlAttributes = @{
+                                        BONValueFromRange(0, 12) : @{
```

Comment:
@plarson please run `clang-format` on this code so that the indentation matches. You can find info on this in the ReadMe. Sorry - should have flagged it sooner.